components/script/dom/webidls/ServiceWorkerContainer.webidl

```diff
@@ -6,9 +6,7 @@
 [Pref="dom.serviceworker.enabled", Exposed=(Window,Worker)]
 interface ServiceWorkerContainer : EventTarget {
   [Unforgeable] readonly attribute ServiceWorker? controller;
-  //[SameObject] readonly attribute Promise<ServiceWorkerRegistration> ready;
-
-  [NewObject, Throws] ServiceWorkerRegistration register(USVString scriptURL, optional RegistrationOptions options);
+  [SameObject] readonly attribute Promise<ServiceWorkerRegistration> ready;
```

Comment:
Your branch is based off of a master commit from Sept 6. You need to be using 2b1a39c2ae14b78e437551d05f0e691a13e5d183 or more recent instead.